stable/database/README.md

```diff
@@ -270,6 +270,9 @@ The following tables list the configurable parameters of the `database` chart an
 | `sm.tolerations` | Tolerations for NuoDB SM | `[]` |
 | `sm.topologySpreadConstraints` | Topology spread constraints for NuoDB SM | `[]` |
 | `sm.readinessTimeoutSeconds` | SM readiness probe timeout, sometimes needs adjusting depending on environment and pod resources | `5` |
+| `sm.storageGroup.enabled` | Enable Table Partitions and Storage Groups (TPSG) for all SMs in this database Helm release | `false` |
```

Comment:
Would it be better if the `te.enablePod` default value was `nil` and we would switch it on if (1) `sm.storageGroup.enabled=true` and (2) `primaryRelease=false`?

Reply:
I'm not sure yet what NuoDB database deployment will look like for the scale-out use case. It may be that you would want to have new TE + SM per SG. This is why I didn't want to code assumptions for `database.te.enablePod` value but rather leave this to the user.

Your proposal will work but `database.te.enablePod=nil` is not consistent with other `enablePod` values and I'm not sure how to set `nil` value for a property in the `values.yaml` (maybe `prop=null`).

Reply:
I meant just commenting out the property in the Helm values, which should allow us to detect that it was explicitly specified.

Reply:
I've implemented this in [e00b6ff](https://github.com/nuodb/nuodb-helm-charts/pull/314/commits/e00b6ff02748525822c723ea715189344c30efdc). It's a bit tricky to check for `nil` though.